package.json

```diff
@@ -2,12 +2,21 @@
   "name": "@zazuko/rdf-vocabularies",
   "version": "2019.5.13",
   "description": "Access most commonly used RDF ontologies/schemas/vocabularies as datasets or n-quads",
-  "main": "index.js",
+  "main": "lib/node/index.js",
+  "module": "lib/es/index.js",
+  "files": [
+    "lib",
+    "ontologies"
+  ],
   "scripts": {
     "fetch": "node fetch.js",
-    "lint": "eslint .",
+    "lint": "eslint . --ext .js,.ts",
     "pretest": "npm run lint",
-    "test": "jest"
+    "test": "jest",
+    "build:node": "tsc --target es2017 --module commonjs --outDir lib/node",
+    "build:web": "tsc --target esnext --module esnext --outDir lib/es; tsc --build tsconfig.web.json",
```

Comment:
The browser build is kind of ugly, where a second `tsc` call replaces the `loadDataset` module in es output